client/post-editor/editor-ground-control/index.jsx

```diff
@@ -351,33 +339,34 @@ export default React.createClass( {
 							{ this.translate( 'Save' ) }
 						</button>
 					}
-					{ this.props.isSaving &&
-						<span className="editor-ground-control__saving">
-							{ this.translate( 'Saving…' ) }
+					{ ! this.isSaveEnabled() &&
+						<span className="editor-ground-control__save-status">
+							{ this.getSaveStatusLabel() }
 						</span>
 					}
 				</div>
-				{
-					this.state.showAdvanceStatus &&
-						<EditPostStatus
-							savedPost={ this.props.savedPost }
-							postDate={ postDate }
-							type={ this.props.type }
-							onSave={ this.props.onSave }
-							onTrashingPost={ this.props.onTrashingPost }
-							onDateChange={ this.setPostDate }
-							site={ this.props.site }>
-						</EditPostStatus>
-				}
 				<div className="editor-ground-control__action-buttons">
-					<button
-						className="editor-ground-control__preview-button button"
+					<Button
+						borderless
+						className="editor-ground-control__preview-button"
 						disabled={ ! this.isPreviewEnabled() }
 						onClick={ this.onPreviewButtonClick }
 						tabIndex={ 4 }
 					>
-						{ this.getPreviewLabel() }
-					</button>
+						<Gridicon icon="visible" /> <span className="editor-ground-control__button-label">{ this.getPreviewLabel() }</span>
+					</Button>
+					<Button
+						borderless
+						className="editor-ground-control__toggle-sidebar"
+						onClick={ this.props.toggleSidebar }
+					>
+						<Gridicon icon="cog" /> <span className="editor-ground-control__button-label">
+							{ this.translate( '{{type/}} Settings', {
```

Comment:
Seems like a good compromise is to manually submit `Post Settings` and `Page Settings` and use `Type: Settings` for other post types. Pushed in https://github.com/Automattic/wp-calypso/pull/11536/commits/909ea032ffd2ff3f6309df02e4235ab09e089221